src/Tablet/Dashboard/Dashboard.js

```diff
@@ -1,17 +1,27 @@
 import React, { useState } from "react";
 import { IonContent } from "@ionic/react";
 
+import Aux from "../../../src/Tablet/hoc/Aux";
```

Comment:
The changes in this file don't seem to do anything.

Also, when you import from `src` you can simply do this `@/Tablet/hoc/Aux` where `@` resolves to `$root/src`